docs/reference/metadata.md

```diff
@@ -119,6 +119,10 @@ with external registration services.
 Persistent identifiers are globally unique in the system, thus you cannot have two records
 with the same system-managed persistent identifier (see also [Metadata > Identifiers](#identifiers-0-n)).
 
+You can add a DOI that is not managed by InvenioRDM by using the provider `external`. You are not able to add `external` DOIs that have a prefix that is configured as part of a different PID provider.
+
+If you utilize the default DataCite PID provider, both the provider and client will be `datacite`. 
```

Comment:
Can you provide more details here? We don't use `"client": "datacite"` when the provider is datacite and it works for us... :thinking:

Reply:
Huh, I'm not sure on this one. In the setup RDM uses "datacite" as the client name https://github.com/inveniosoftware/invenio-rdm-records/blob/ab4390dbcf264b3dc63140e0ad6f1e541747f4b7/invenio_rdm_records/config.py#L276, but I guess it works fine without it as long as you only have one instance of "datacite". I'm not sure what best practice we want to recommend for the documentation.

Reply:
If we remove line 124 and the change on line 140 are the docs still providing the info that was missing? If yes, let's go for that.

Reply:
I'm fine removing line 124, if we leave the change in line 140. Having 'zenodo' as a client example doesn't make sense, unless there is an explanation for how it should be configured in a standard RDM install.

Reply:
:+1: For line 140, something like `"client": "<your client name>"`?

Reply:
I've pulled 124. I think datacite is better for 140 because that is the default. You wouldn't set a different client name unless you're doing some serious customization.

Reply:
The idea behind `client` in a provider is to be able to have on one provider (e.g. DataCite) multiple clients (one client per prefix). But at the moment I don't think anybody is actively using that functionality. 

The example had `zenodo` due to the fact that we used the config we had there. So `datacite` makes more sense. Thanks both of you!